charts/astronomer/templates/_helpers.yaml

```diff
@@ -139,6 +135,26 @@
 {{- end }}
 {{- end }}
 
+{{ define "houston.bootstrapperServiceAccount" -}}
+{{ printf "%s-houston-bootstrapper" .Release.Name }}
+{{- end }}
+
+{{ define "configSyncer.ServiceAccount" -}}
+{{- if .Values.configSyncer.serviceAccount.name -}}
+{{ printf "%s-%s" .Release.Name .Values.configSyncer.serviceAccount.name }}
+{{- else -}}
+{{ printf "%s-configSyncer" .Release.Name }}
+{{- end }}
+{{- end }}
+
+{{ define "commander.ServiceAccount" -}}
+{{- if .Values.commander.serviceAccount.name -}}
+{{ printf "%s-%s" .Release.Name .Values.commander.serviceAccount.name }}
+{{- else -}}
+{{ printf "%s-commander" .Release.Name }}
+{{- end }}
+{{- end }}
```

Comment:
Would it be better to have the boostrapperServiceAccount use the exact same logic as the other ones, if for no other reason than to be consistent? CC @pgvishnuram

Reply:
this was changed to 
`{{ define "houston.bootstrapperServiceAccount" -}}
{{- if .Values.houston.serviceAccount.create -}}
{{ default (printf "%s-houston-bootstrapper" .Release.Name ) .Values.houston.serviceAccount.name }}
{{- else -}}
    {{ default "default" .Values.houston.serviceAccount.name }}
{{- end }}
{{- end }}` by @pgvishnuram